media-gfx/rawtherapee/rawtherapee-5.6.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils flag-o-matic toolchain-funcs xdg-utils
+
+MY_P=${P/_rc/-rc}
+DESCRIPTION="A powerful cross-platform raw image processing program"
+HOMEPAGE="http://www.rawtherapee.com/"
```

Comment:
I had already changed this to `https` on latest ebuild, is this based on an older ebuild somehow?

Reply:
yes, I had changes reqested at the time we pushed rawtherapee 5.5 and it didn't came to mind to also compare the in-tree ebuild to the one I have in overlay. Note taken for 5.7